.rubocop.yml

```diff
@@ -1,6 +1,18 @@
 Metrics/LineLength:
   Max: 100
 
+Metrics/AbcSize:
+  Max: 20
+
+Metrics/CyclomaticComplexity:
+  Max: 10
+
+Metrics/PerceivedComplexity:
+  Max: 10
+
+Style/DoubleNegation:
```

Comment:
This was already disabled in hammer feature, could you rebase please?

Reply:
Thanks, rebased